src/clusterfuzz/_internal/tests/core/bot/fuzzers/centipede/centipede_engine_test.py

```diff
@@ -369,6 +368,7 @@ def test_minimize_corpus(self):
 
     self.assertTrue(result)
     self.assertEqual(len(os.listdir(minimized_corpus)), 3)
+    self.assertEqual(len(os.listdir(minimized_corpus)), 4)
```

Comment:
I think it will work, but I do need to remove the `skip`: I discarded that line while removing some debug statements.
I explained why it should work in the issue you created: https://github.com/google/clusterfuzz/issues/3622

Let me remove the `skip` and test again.

Reply:
Also removed the old line (I suppose I incorrectly discarded the change either).

Reply:
Would it make sense to change this to assertLessEqual to 4?  Corpus Minimization is not necessarily deterministic, and anything that gets <= 4 should suffice here.

Reply:
> Would it make sense to change this to assertLessEqual to 4? Corpus Minimization is not necessarily deterministic, and anything that gets <= 4 should suffice here.

Oh, that's the reason!
I was confused about the cloud build error and wonder why it is 2 now.
Fixing this now.

BTW, Is the result non-deterministic because we process them in different orders everytime?

Reply:
> Would it make sense to change this to assertLessEqual to 4? Corpus Minimization is not necessarily deterministic, and anything that gets <= 4 should suffice here.

I changed it to strictly fewer than 6 to be more robust.
We started with 7 inputs (including 1 crasher, which will not go into the minimized corpus), so strictly fewer than 6 shows at least one testcase is minimized.

Reply:
Thanks! Not sure on the exact reasons, but I would guess processing order is one of them.